Userland/Libraries/LibChess/UCICommand.cpp

```diff
@@ -323,16 +324,181 @@ ErrorOr<String> BestMoveCommand::to_string() const
     return builder.to_string();
 }
 
-ErrorOr<NonnullOwnPtr<InfoCommand>> InfoCommand::from_string([[maybe_unused]] StringView command)
+ErrorOr<NonnullOwnPtr<InfoCommand>> InfoCommand::from_string(StringView command)
 {
-    // FIXME: Implement this.
-    VERIFY_NOT_REACHED();
+    auto tokens = command.split_view(' ');
+    VERIFY(tokens[0] == "info");
+
+    auto info_command = TRY(adopt_nonnull_own_or_enomem(new (nothrow) InfoCommand));
```

Comment:
Can't you use `try_make` here?
When it comes to Serenity's codebase, consider `new` as a code smell.

Reply:
I copied the pattern that was recently added elsewhere. It looks like `try_make` does exactly the same thing. 
I've changed it :^)